ftl/python/builder_test.py

```diff
@@ -58,9 +59,11 @@ def setUp(self, mock_from):
         args.tar_base_image_path = None
         self.builder = builder.Python(self.ctx, args)
         self.interpreter_builder = layer_builder.InterpreterLayerBuilder(
-            self.builder._venv_dir, self.builder._python_cmd,
+            ftl_util.gen_tmp_dir(constants.VENV_DIR[1:]),
```

Comment:
can you do a `constants.VENV_DIR.replace('/', '')` instead? and also put a comment here explaining why you're doing it this way?